templates/search/blocks/search_filters.html

```diff
@@ -113,7 +135,7 @@ <h4 class="search-filters__accordion-section-heading" id="record_opening_date">R
             {% endif %}
         </div>
 
-        {{ 'filter_keyword collection topic level closure opening_start_date opening_end_date catalogue_source held_by type country location'|include_hidden_fields:form }}
+        {{ 'catalogue-filters-form'|include_hidden_fields:form }}
```

Comment:
@TNA-Allan A custom filter just isn't the right device here. This should really be [simple tag](https://docs.djangoproject.com/en/4.2/howto/custom-template-tags/#simple-tags), and we need to find a way for these values to be specified on the form instance, so that they aren't hardcoded in the template.

Reply:
> A custom filter just isn't the right device here. This should really be [simple tag](https://docs.djangoproject.com/en/4.2/howto/custom-template-tags/#simple-tags), and we need to find a way for these values to be specified on the form instance, so that they aren't hardcoded in the template

Perhaps would it be bound field methods, but I'm not sure how to go about it with the complexities for this specific page with multiple forms and their behaviours (and upcoming tickets to reflect that).

Reply:
`{% render_form_fields_as_hidden form include='sort_by field_1 field_2' %}` would be a nicer interface here.

Reply:
see comment below for `render_form_fields_as_hidden`